app/controllers/home_controller.rb

```diff
@@ -1,15 +1,11 @@
 class HomeController < ApplicationController
   def home
-    if(params[:re_login])
-      redirect_to user_omniauth_authorize_path(params[:provider])
```

Comment:
OK, so this won't change logging into Lara (as an author say) via the 'login from <hostname>' links?